src/app/zap-templates/zcl/data-model/chip/scene.xml

```diff
@@ -128,36 +128,36 @@ limitations under the License.
       <description>
         Allows a scene to be retrieved using a finer scene transition time than the ViewScene command
       </description>
-      <arg name="GroupId" type="group_id"/>
-      <arg name="SceneId" type="INT8U"/>
+      <arg name="GroupID" type="group_id"/>
+      <arg name="SceneID" type="INT8U"/>
     </command>
     <command source="client" code="0x42" name="CopyScene" response="CopySceneResponse" isFabricScoped="true" optional="true" cli="chip scenes copy">
       <description>
         Allows a client to efficiently copy scenes from one group/scene identifier pair to another group/scene identifier pair.
       </description>
       <arg name="Mode" type="ScenesCopyMode"/>
-      <arg name="GroupIdFrom" type="group_id"/>
-      <arg name="SceneIdFrom" type="INT8U"/>
-      <arg name="GroupIdTo" type="group_id"/>
-      <arg name="SceneIdTo" type="INT8U"/>
+      <arg name="GroupIdentifierFrom" type="group_id"/>
+      <arg name="SceneIdentifierFrom" type="INT8U"/>
+      <arg name="GroupIdentifierTo" type="group_id"/>
+      <arg name="SceneIdentifierTo" type="INT8U"/>
     </command>
     
     <command source="server" code="0x00" name="AddSceneResponse" optional="false" disableDefaultResponse="true">
       <description>
         The command is generated in response to a received unicast AddScene command,
       </description>
       <arg name="Status" type="ENUM8"/>
-      <arg name="GroupId" type="group_id"/>
-      <arg name="SceneId" type="INT8U"/>
+      <arg name="GroupID" type="group_id"/>
+      <arg name="SceneID" type="INT8U"/>
     </command>
     
     <command source="server" code="0x01" name="ViewSceneResponse" optional="false" disableDefaultResponse="true">
       <description>
         The command is generated in response to a received unicast ViewScene command 
       </description>
       <arg name="Status" type="ENUM8"/>
-      <arg name="GroupId" type="group_id"/>
-      <arg name="SceneId" type="INT8U"/>
+      <arg name="GroupID" type="group_id"/>
+      <arg name="SceneID" type="INT8U"/>
```

Comment:
should SceneName below have a length=16 constraint? Or does that not matter?

Reply:
We do nothing with constraints for command fields right now.  So does not matter in practice.